src/easylink/rule.py

```diff
@@ -201,12 +206,16 @@ def _build_resources(self) -> str:
 
     def _build_shell_cmd(self) -> str:
         """Builds the shell command portion of the rule."""
-        # TODO: handle multiple wildcards
-        # output_paths = ",".join(self.output)
-        # wildcards_subdir = "/".join([f"{{wildcards.{wc}}}" for wc in self.wildcards])
-        # and then in shell cmd: export DUMMY_CONTAINER_OUTPUT_PATHS={output_paths}/{wildcards_subdir}
+        # TODO [MIC-5877]: handle multiple wildcards, e.g.
```

Comment:
Is that really the right ticket? https://jira.ihme.washington.edu/browse/MIC-5877

If so, could use more explanation of why